apps/console/src/features/users/components/edit-user.tsx

```diff
@@ -83,12 +86,28 @@ export const EditUser: FunctionComponent<EditUserPropsInterface> = (
         (state: AppState) => state?.config?.ui?.isGroupAndRoleSeparationEnabled);
 
     const [ isReadOnly, setReadOnly ] = useState<boolean>(false);
+    const [ isSuperAdmin, setIsSuperAdmin ] = useState<boolean>(false); 
+    const [ isSelectedSuperAdmin, setIsSelectedSuperAdmin ] = useState<boolean>(false); 
+    const [ 
+        isSuperAdminIdentifierFetchRequestLoading, 
+        setIsSuperAdminIdentifierFetchRequestLoading 
+    ] = useState<boolean>(false);
+    const [ hideTermination, setHideTermination ] = useState<boolean>(false);
+    const [ user, setUser ] = useState<ProfileInfoInterface>(selectedUser);
 
     useEffect(() => {
-        if(!user) {
+    /**
```

Comment:
Check the formatting here. Better to add a `//` comment here.

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/3007/commits/6f0195b5af403dc62ed0ee4af24016368ccbf155